Lønn v2 (#4)
apparatno/company: lonnsmodell.md

## Prinsipper

Hos Apparat skal du ha en god og rettferdig lønn. Vi mener det er viktig at lønnen passer til deg, og at du skal ha noe konkret å strekke deg etter.

> Lønn består av fastlønn og provisjon, og følger en fastsatt, trinnvis økning. Fastlønn øker med 5000kr og provisjon øker med 1% hvert år. Man går opp minst 1 trinn i året.

Reviewer: Dette gjelder vel kun IT/produksjon? Bør det være mer tydelig kanskje?
```suggestion
Lønn består av fastlønn og provisjon, og følger en fastsatt, trinnvis økning. Fastlønn øker med 5 000 kr og provisjon øker med 1% hvert år. Man går opp minst 1 trinn i året.
```

Provisjonsleddet gjelder fra første fakturerte krone - ingen terskel. Vi synes det er mest rettferdig at du tjener penger når selskapet gjør det. På den måten deler vi på det hele tiden. I tillegg gir det deg enkel oversikt, og du kan lett regne ut lønnen din hver måned basert på hvor mange timer du har jobbet.

## Våre modeller

### IT

> Dette er modellen for programvareutviklere

Reviewer: Hva med andre typer roller vi evt ansetter i fremtiden, f.eks. testere eller prosjektledere?

Reviewer 2: Enig, jeg foreslår IT-konsulenter, hvor alle roller inngår. Og så bør vi ordne noe med medie-greiene, men det kreves litt mer arbeid her.

> Dette er modellen for programvareutviklere

Reviewer 2: ```suggestion
Dette er modellen for IT-konsulenter
```

Reviewer 2: om ikke `IT-konsulenter` fungerer, så `ansatte som jobber med IT`?

> > [Les mer om hvordan IT-modellen fungerer her](lonnsmodell-produksjon.md).

Reviewer 2: ```suggestion
> [Les mer om hvordan lønnen for IT-konsulenter fungerer her](lonnsmodell-produksjon.md).
```
